user_guide_src/source/changelogs/v4.4.0.rst

```diff
@@ -59,6 +59,10 @@ Forge
 Others
 ------
 
+- **Database:** Added the ``numberNative`` attribute to the Database Config for MySQLi to keep the variable type obtained after SQL Query consistent with the type set in the database.
+  See `configuration.rst <https://github.com/codeigniter4/CodeIgniter4/blob/4.4/user_guide_src/source/database/configuration.rst>`_  
```

Comment:
This is a link to the GitHub repository.
We need to link to the User Guide itself, and it is not only by HTML but also EPUB.
So we need to use links of Sphinx.

```suggestion
  See :ref:`Database Configuration <database-configuration-explanation-of-values>`.
```

and add  the following:
```diff
--- a/user_guide_src/source/database/configuration.rst
+++ b/user_guide_src/source/database/configuration.rst
@@ -104,6 +104,8 @@ default group's configuration settings. The values should be name following this
     database.default.password = '';
     database.default.database = 'ci4';
 
+.. _database-configuration-explanation-of-values:
+
 **********************
 Explanation of Values:
 **********************
```

Reply:
Before push commit, I would like to ask, is this the desired modification?
![image](https://user-images.githubusercontent.com/62149120/219931840-c9a73984-0e41-494b-a77d-1034be07a598.png)

Reply:
No.
```

.. _database-configuration-explanation-of-values:

**********************
Explanation of Values:
**********************
```

Reply:
Does it look like this?
![image](https://user-images.githubusercontent.com/62149120/219932107-7037b262-20e3-4813-8d40-9e1ce0d112e5.png)

Reply:
Yes!

Reply:
Thank you, I have corrected these descriptions.